en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,260 @@
+    
+    Save the information.
+    
+2. To respond with the JSON payload, add a `Respond` statement.
 
-        ![Add Respond Statement](../assets/img/services/add-respond-statement.png){.cInlineImage-half}
+    Click the last **+** icon in the low-code diagram and click **Respond**.
+    
+    In the **Respond Expression** field, enter `payload`.
     
-    2. In the **Respond Expression** field, enter a response and then click **Save**.
+    Save the information.
+    
+Now you have completed designing the `CovidStatus` service.
 
-Now you have completed designing the service. It looks as follows:
+The low-code diagram looks as follows:
 
-- In the low-code view
+![Completed low-code diagram](../assets/img/services/completed-low-code-diagram.png){.cInlineImage-bordered}
 
-  ![No Code View](../assets/img/services/choreo-service-low-code-view.png){.cInlineImage-half}
+The code view looks as follows:
 
-- In the code view
+```ballerina
+import ballerinax/worldbank;
+import ballerinax/covid19;
+import ballerina/http;
 
-  ![Code View](../assets/img/services/choreo-service-code-view.png){.cInlineImage-full}
+service / on new http:Listener(8090) {
+    resource function get stats/[string country](http:Caller caller, http:Request request) returns error? {
+        covid19:Client covid19Client = check new ();
+        covid19:CovidCountry statusByCountry = check covid19Client->getStatusByCountry(country);
+        var totalCases = statusByCountry?.cases ?: 0d;
+        worldbank:Client worldBankClient = check new ();
+        worldbank:CountryPopulationArr? populationByCountry = check worldBankClient->getPopulationByCountry(country, 
+        "2019", format = "json");
+        var population = 
+        (populationByCountry is worldbank:CountryPopulationArr ? populationByCountry[0]?.value ?: 0 : 0) / 1000000;
 
-    
-To validate the service, click **Run & Test**. The following is logged to indicate that you have successfully started the service.
+        var totalCasesPerMillion = totalCases / population;
+        var payload = {
+            country: country,
+            totalCasesPerMillion: totalCasesPerMillion
+        };
 
-![Service Started Log](../assets/img/services/service-started-notification.png){.cInlineImage-half}
-    
-Now you are ready to test the service.
-   
-## Step 3: Test the service
+        check caller->respond(payload);
+    }
+}
+```
+
+## Step 6: Test the service
+
+To test the `CovidStatus` service you created, follow the procedure below:
 
-To test the `covid-stats` service, follow this procedure:
+1. Click **Run & Test**.
 
-1. Click the **Test** icon in the left pane.
+    The following logs appear to indicate that the service has successfully started.
+    
+    ```
+    Starting application...
+    [ballerina/http] started HTTP/WS listener 0.0.0.0:8090
+    ```
+   
+2. Click the **Test** icon in the left pane.
 
     ![Test Icon](../assets/img/services/test-icon.png){.cInlineImage-bordered}
```

Comment:
Need to follow sentence-style for alt text.

Reply:
Corrected

Reply:
Corrected